_posts/2023-10-23-kubeflow-1.8-release.md

```diff
@@ -0,0 +1,374 @@
+* Adding an end-to-end example for deploying a large language model on KServe ([2836](https://github.com/kserve/kserve/pull/2836))
+
+More information on the KServe 0.11 release can be found in the [release notes](https://github.com/kserve/kserve/releases/tag/v0.11.0/)
+
+### Simplifying Storage for ML Workflows
+
+Kubeflow’s web apps enhancements expose valuable information to data scientists, which simplifies advanced workflows.  For example, Kubernetes storage management, especially for persistent volumes, is critical for rapid data science iteration but can require complex, manual configuration.  The newly introduced PVC Viewer makes storage management easy for end users and eliminates the need to learn low-level CLI commands.    Users can click and drag to add/remove files and file directories from president volumes.  The volumes can easily be attached to MLOps objects, like notebooks, which makes experimenting with different data sets faster.
+
+
+![Kubeflow user using pvc-viewer](/images/2023-10-23-kubeflow-1.8-release/pvc-viewer-demo.gif)
+
+
+### Support for ARM Machines 
+
+
+Kubeflow 1.8 brings forward a significant upgrade [(#7343)](https://github.com/kubeflow/kubeflow/pull/7342) by officially introducing support for ARM architectures. This move ensures that developers can leverage Kubeflow's powerful machine learning capabilities on a wider variety of devices and platforms, including the likes of Apple Silicon and a multitude of IoT devices. This expansion highlights Kubeflow's adaptability, ensuring seamless machine learning workflows across various hardware architectures.
+
+
+### Security - Rootless Kubeflow Updates and Fixes
+
+In our ongoing commitment to bolstering security in the project, version 1.8 brings both vital security fixes and innovative features. A notable endeavor by the Security has been the development of features that lay the groundwork for a [rootless Kubeflow #2455](https://github.com/kubeflow/manifests/pull/2455) in the future. v1.8 offers users the flexibility to run Kubeflow either with the optional istio-cni or without it. A selection of these features and bug fixes include:
+
+* Fix performance issue within a mysql request that could cause a denial of service ([#9680](https://github.com/kubeflow/pipelines/pull/9680))
+* Fix issue on which Profile controller and KFAM allow unauthenticated in-cluster traffic([#7032](https://github.com/kubeflow/kubeflow/pull/7032))
+* Adding oauth2-proxy as optional alternative to oidc-authservice ([#2409](https://github.com/kubeflow/manifests/pull/2409))
+
+
+Future versions will continue to refine this rootless functionality, along with plans to make istio-cni the default option along with istio ambient mesh and incorporating [Pod Security Standards](https://github.com/kubeflow/kubeflow/pull/7325). Please join the [Security Working Group](https://kubeflow.slack.com/archives/C04CF10FJ2D)
+
+### Notebooks
```

Comment:
Given that Kubeflow Notebooks is the second most widely used component of Kubeflow (according to the surveys), I think we should move this section to be second, directly under `### Kubeflow Pipelines`

Also, please update the title to: `### Kubeflow Notebooks`